README.md

```diff
@@ -1,33 +1,34 @@
 # testing-farm-as-github-action
 
-Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
+Testing Farm as GitHub Action is a GitHub Action for executing tests on the Testing Farm infrastructure.
 
-This action runs on [composite](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action).
-Before a calling this GitHub Action, you must first clone your project by e.g. 
-by GitHub Action [Checkout V2](https://github.com/actions/checkout).
+The tests to run are to be described with a [`fmf` plan](https://tmt.readthedocs.io/en/latest/spec.html) by the user of this GitHub Action.
+Pull Request status is automatically updated after the tests are executed.
```

Comment:
This is true but can be changed by setting value `update_pull_request_status` to false.

Reply:
Thanks for catching this, I will correct it.